application/comit_node/build.rs

```diff
@@ -10,12 +8,20 @@ use std::{
 };
 
 fn main() -> std::io::Result<()> {
-    compile("./src/swap_protocols/rfc003/ethereum/contract_templates/ether_contract.asm")?;
-    compile("./src/swap_protocols/rfc003/ethereum/contract_templates/ether_deploy_header.asm")?;
-    compile("./src/swap_protocols/rfc003/ethereum/contract_templates/erc20_contract.asm")?;
-    compile("./src/swap_protocols/rfc003/ethereum/contract_templates/erc20_deploy_header.asm")?;
-
-    Ok(())
+    match var("COMPILE_EVM").as_ref().map(|s| s.as_str()) {
```

Comment:
:x: Actually, the way `cargo` build scripts work. This should be done automatically. If you look at line 74 of this file, you will see that we emit a `re-run if changed` instruction to cargo.

That means, as long as the `.asm` files don't change, the build-script won't run. If you change them, the cargo will detect it and re-run the build-script.

Hence, I don't think there is a need for the additional `cargo-make` task. `cargo` should be able to handle all of that by it's own :)

Reply:
Except for the first time you build, you would need to have `docker` installed to allow `build.rs` to emit such line and not build `again`.
We are trying to avoid to build at all (ie, no docker for a fresh checkout).

Reply:
IMO, the complexity introduced (in this PR) is not worth the "benefit" of not needing docker.